model/project_ref.go

```diff
@@ -247,16 +247,55 @@ func (p *ProjectRef) Add(creator *user.DBUser) error {
 	return p.AddPermissions(creator)
 }
 
+func (p *ProjectRef) GetRepoId() string {
+	return fmt.Sprintf("%s_%s", p.Owner, p.Repo)
+}
+
+func (p *ProjectRef) HasEditPermission(u *user.DBUser) bool {
+	if utility.StringSliceContains(p.Admins, u.Username()) {
```

Comment:
Is this still something that we need to check? It looks like when we give admins access, we also give them these permissions with UpdateAdminRoles.

Reply:
we shouldn't need to check this. It's maybe a useful sanity check but we should assume that admins also have the right permissions